src/h3lib/include/coordijk.h

```diff
@@ -54,22 +54,27 @@ static const CoordIJK UNIT_VECS[] = {
     {1, 1, 0}   // direction 6
 };
 
-// H3 digit representing ijk+ axes direction
-// see also unitVecs in coordijk.c
-/** H3 digit in center */
-#define CENTER_DIGIT 0
-/** H3 digit in k-axes direction */
-#define K_AXES_DIGIT 1
-/** H3 digit in j-axes direction */
-#define J_AXES_DIGIT 2
-/** H3 digit in j == k direction */
-#define JK_AXES_DIGIT (K_AXES_DIGIT | J_AXES_DIGIT) /* 3 */
-/** H3 digit in i-axes direction */
-#define I_AXES_DIGIT 4
-/** H3 digit in i == k direction */
-#define IK_AXES_DIGIT (I_AXES_DIGIT | K_AXES_DIGIT) /* 5 */
-/** H3 digit in i == j direction */
-#define IJ_AXES_DIGIT (I_AXES_DIGIT | J_AXES_DIGIT) /* 6 */
+/** @brief H3 digit representing ijk+ axes direction.
+ * Values will be within the lowest 3 bits of an integer.
+ */
+typedef enum {
+    /** H3 digit in center */
+    CENTER_DIGIT = 0,
+    /** H3 digit in k-axes direction */
+    K_AXES_DIGIT = 1,
+    /** H3 digit in j-axes direction */
+    J_AXES_DIGIT = 2,
+    /** H3 digit in j == k direction */
+    JK_AXES_DIGIT = K_AXES_DIGIT | J_AXES_DIGIT, /* 3 */
```

Comment:
tiniest nit ever: swap j & k OR to `J_AXES_DIGIT | K_AXES_DIGIT` s.t. `JK_... = J_... | K_...` :P